content/en/platform/corda/4.8/open-source/generating-a-node-cordform.md

```diff
@@ -11,17 +11,17 @@ menu:
 tags:
 - cordform
 - node
-title: Cordform plug-in
+title: Cordform tasks
 ---
 
-# Cordform plug-in
+# Cordform tasks
 
-Corda provides two `gradle` plug-ins - `Cordform` and `Dockerform`. They both allow you to run tasks that automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Cordform plug-in. Visit the [Dockerform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-dockerform.md) page for Dockerform configuration options.
+Corda's `cordformation` Gradle plugin provides the `Cordform` and `Dockerform` tasks. They both allow you to automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Cordform tasks. Visit the [Dockerform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-dockerform.md) page for Dockerform configuration options.
```

Comment:
"the operation of the Cordform task" (singular).

Reply:
updated